frontend/src/MuiForms/CustomTextInput.tsx

```diff
@@ -12,40 +14,48 @@ interface EntitySelectorProps {
   InputProps?: any
 }
 
-export default function EntitySelector(props: EntitySelectorProps) {
+export default function CustomTextInput(props: CustomTextInputProps) {
   const { onChange, value: currentValue, label, formContext } = props
```

Comment:
Not a change that you've made, but there's no need to change `value` to `currentValue` here. It just hinders readability by renaming the prop unnecessarily. However, it's up to you if you want to change it or not in this PR.